subsys/spm/Kconfig

```diff
@@ -131,6 +131,13 @@ config SPM_NRF_SAADC_NS
 	bool "SAADC is Non-Secure"
 	default y
 
+config SPM_NRF_PWM0_NS
+	bool "PWM0 is Non-Secure"
+	default y
+
+config SPM_NRF_PWM1_NS
+	bool "PWM1 is Non-Secure"
+
```

Comment:
All peripherals which can be configured as non-secure, should have non-secure as default.
Unless there is a valid reason for not doing so (e.g. UARTE2 reserved for mcuboot usage).